src/main/java/com/exasol/bucketfs/WriteEnabledBucket.java

```diff
@@ -88,7 +88,7 @@ protected BucketAccessException createUploadInterruptedException(final URI uri)
                 messageBuilder("E-BFSJ-6").message("Interrupted trying to upload {{URI}}.", uri).toString());
     }
 
-    private void recordUploadInHistory(final String pathInBucket) {
+    protected void recordUploadInHistory(final String pathInBucket) {
```

Comment:
Should we add Javadocs for protected methods?

Reply:
Fixed.